include/swift/Runtime/RuntimeFunctions.def

```diff
@@ -1222,6 +1222,14 @@ FUNCTION(RegisterTypeMetadataRecords,
          ARGS(TypeMetadataRecordPtrTy, TypeMetadataRecordPtrTy),
          ATTRS(NoUnwind))
 
+// void swift_registerTypeFields(const FieldDescriptor *begin,
+//                               const FieldDescriptor *end)
+FUNCTION(RegisterTypeFields,
```

Comment:
This file is only used for functions called from IRGen, so I don't think this is necessary.

Reply:
Gotcha! Will remove.